packages/eas-cli/src/commands/update/index.ts

```diff
@@ -503,3 +529,7 @@ export default class UpdatePublish extends EasCommand {
     };
   }
 }
+
+function setEquals<T>(set1: Set<T>, set2: Set<T>): boolean {
```

Comment:
use https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/utils/expodash/areSetsEqual.ts

Reply:
I unresolved the comment, if there is a reason why it was resolved, let me know. I guess you might have not pushed the changes to the PR yet :)

Reply:
Sorry, I generally use these as a todo list to check off while I address comments. The second comment took a lot longer to address.